avaframe/com1DFAPy/com1DFA.py

```diff
@@ -798,12 +819,25 @@ def initializeMassEnt(dem, logName, entLine):
         entrainmentArea = entLine['Name']
         log.info('Entrainment area: %s' % (entrainmentArea))
         entrMassRaster = prepareArea(entLine, dem)
+        areaInfo['entrainment'] = 'Yes'
+    elif ('ent' in logName) and entLine:
+        entrainmentArea = entLine['Name']
+        log.info('Entrainment area: %s' % (entrainmentArea))
+        entrMassRaster = prepareArea(entLine, dem)
+        areaInfo['entrainment'] = 'Yes'
+    elif ('all' in logName) and entLine:
+        entrainmentArea = entLine['Name']
+        log.info('Entrainment area: %s' % (entrainmentArea))
+        entrMassRaster = prepareArea(entLine, dem)
+        areaInfo['entrainment'] = 'Yes'
```

Comment:
This could be reduced to only one if no? you do the same in every case

Reply:
done